src/components/PageLoading.vue

```diff
@@ -1,15 +1,15 @@
 <template>
-  <div>
+  <div class="space-y-3">
     <div
-      class="bg-skin-text rounded-md animate-pulse-fast mb-3"
+      class="bg-skin-text rounded-md animate-pulse-fast"
       style="width: 100%; height: 34px"
     />
     <div
-      class="bg-skin-text rounded-md animate-pulse-fast mb-3"
+      class="bg-skin-text rounded-md animate-pulse-fast"
       style="width: 40%; height: 34px"
     />
     <div
-      class="bg-skin-text rounded-md animate-pulse-fast mb-4"
```

Comment:
This is `4` not `3`

Reply:
@samuveth why we need mb-4 for last element?